### PR TITLE
Sending Emails

### PR DESCRIPTION
**Pull request by miritih**
**Problem we are solving for**
App was not able to send emails alerts

**Solutions**
Add settings to allow djmail to send alerts via Gmail

### CODE REVIEW

**Review by simiyu1 — APPROVED** [review]
I like the fact that you used tls security over ssl.

File: hc/settings.py [hunk 1]
```diff
@@ -136,7 +136,15 @@
 )
 COMPRESS_OFFLINE = True
 
+#send emails using Gmail.
 EMAIL_BACKEND = "djmail.backends.default.EmailBackend"
+DJMAIL_REAL_BACKEND = 'djmail.backends.async.EmailBackend'
+EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
+EMAIL_HOST = 'smtp.gmail.com'
```
Comment (miritih) [inline hc/settings.py]: The code is now updated, Check commit 997ce1b0bf2cf0279211b729ae91fbd60bfe5176 to review

[the remark this answers is not in the document]
Comment (miritih) [reply hc/settings.py]: The code is now updated, Check commit 997ce1b0bf2cf0279211b729ae91fbd60bfe5176 to review

**Review by thoth-ky — APPROVED** [review]
good work

**Review by loicemeyo — APPROVED** [review]

**Pull request merged**